### ISSUE TITLE
Feature: CSS open tracking for email clients, specifically Outlook

### ISSUE THREAD

**Issue by Static-Flow** [gophish/gophish#1354]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: 0.7.1

Brief description of the issue: 

So I have a solution for tracking Outlook email clients using CSS as a fallback for or addition to tracking pixels which would solve #771 and #263. I have the code already written and it seems like it should work. The issue I'm running into is actually seeing my changes work. From what I can tell the project builds from the remote source of the official gophish repo so even if I fork the repo any changes I make to the Go files will not show. 

I would love to add to the functionality of this great tool I just need some guidance on how to create a development workspace to test it before starting the PR process.

**Comment by Static-Flow** [comment]
I realize now what I was doing wrong. It doesn't write it to whatever directory you're in it writes it to GOPATH. I'll be attaching a PR to this issue shortly but would still appreciate any guidance on PR procedures.

**Comment by pranavperfect** [comment]
Any updates to this?
